### ISSUE TITLE
Auto-restart training on multi GPUs after reached iterations threshold.

### ISSUE THREAD

**Issue by folkien** [AlexeyAB/darknet#7905]
For Feature-request:
    * describe your feature as detailed as possible

To improve training speed (reduce training hours) could be added multi-gpu auto restart feature. User could for example define in .cfg file threshold value after `how many iterations` training should be restarted on multi gpu. 

At the moment we have to manually start new training on single GPU, and manually stop it and continue  on multi gpu. According to @AlexeyAB  description

> 
> How to train with multi-GPU
> 
>     Train it first on 1 GPU for like 1000 iterations: darknet.exe detector train cfg/coco.data cfg/yolov4.cfg yolov4.conv.137
> 
>     Then stop and by using partially-trained model /backup/yolov4_1000.weights run training with multigpu (up to 4 GPUs): darknet.exe detector train cfg/coco.data cfg/yolov4.cfg /backup/yolov4_1000.weights -gpus 0,1,2,3
> 
> If you get a Nan, then for some datasets better to decrease learning rate, for 4 GPUs set learning_rate = 0,00065 (i.e. learning_rate = 0.00261 / GPUs). In this case also increase 4x times burn_in = in your cfg-file. I.e. use burn_in = 4000 instead of 1000.
